cloud/services/internalloadbalancers/internalloadbalancers.go

```diff
@@ -43,7 +43,7 @@ func (s *Service) Reconcile(ctx context.Context, spec interface{}) error {
 	if !ok {
 		return errors.New("invalid internal load balancer specification")
 	}
-	klog.V(2).Infof("creating internal load balancer %s", internalLBSpec.Name)
+	s.Scope.Logger.V(2).Info("creating internal load balancer", "load balancer", internalLBSpec.Name)
```

Comment:
nit: below you used `"internal lb"` for the internal load balancer, let's be consistent and use one or the other everywhere.

Reply:
done, thanks!